{
 "pytest_github_actions_annotate_failures": {
  "data": {},
  "type": "module"
 },
 "pytest_github_actions_annotate_failures.plugin": {
  "data": {},
  "type": "module"
 },
 "pytest_github_actions_annotate_failures.plugin.OrderedDict": {
  "data": {
   "shadows": "collections.OrderedDict"
  },
  "type": "import"
 },
 "pytest_github_actions_annotate_failures.plugin._error_workflow_command": {
  "data": {
   "lineno": 55,
   "symbols_in_volume": [
    "_escape",
    "collections.OrderedDict"
   ]
  },
  "type": "function"
 },
 "pytest_github_actions_annotate_failures.plugin._escape": {
  "data": {
   "lineno": 70
  },
  "type": "function"
 },
 "pytest_github_actions_annotate_failures.plugin.os": {
  "data": {
   "shadows": "os"
  },
  "type": "import"
 },
 "pytest_github_actions_annotate_failures.plugin.print_function": {
  "data": {
   "shadows": "__future__.print_function"
  },
  "type": "import"
 },
 "pytest_github_actions_annotate_failures.plugin.pytest": {
  "data": {
   "shadows": "pytest"
  },
  "type": "import"
 },
 "pytest_github_actions_annotate_failures.plugin.pytest_runtest_makereport": {
  "data": {
   "lineno": 14,
   "symbols_in_volume": [
    "AttributeError",
    "_error_workflow_command",
    "print",
    "pytest_github_actions_annotate_failures.plugin.os",
    "pytest_github_actions_annotate_failures.plugin.pytest"
   ]
  },
  "type": "function"
 }
}